### PR TITLE
Passwordless PKI mode

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

File: doc/server.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ core.https\_allowed\_origin         | string    | global    | -         | -
 core.proxy\_https                   | string    | global    | -         | -                                 | https proxy to use, if any (falls back to HTTPS\_PROXY environment variable)
 core.proxy\_http                    | string    | global    | -         | -                                 | http proxy to use, if any (falls back to HTTP\_PROXY environment variable)
 core.proxy\_ignore\_hosts           | string    | global    | -         | -                                 | hosts which don't need the proxy for use (similar format to NO\_PROXY, e.g. 1.2.3.4,1.2.3.5, falls back to NO\_PROXY environment variable)
+core.trust\_ca\_certificates        | boolean   | global    | -         | -                                 | Whether to check for client certificates trusted by the CA
```
Comment (stgraber) [inline doc/server.md]: The description probably should be clearer, something like `Whether to automatically trust clients signed by the CA`

File: lxd/main_init.go [hunk 2]
```diff
@@ -45,7 +46,7 @@ func (c *cmdInit) Command() *cobra.Command {
 	cmd.Example = `  init --preseed
   init --auto [--network-address=IP] [--network-port=8443] [--storage-backend=dir]
               [--storage-create-device=DEVICE] [--storage-create-loop=SIZE]
-              [--storage-pool=POOL] [--trust-password=PASSWORD]
+              [--storage-pool=POOL] [--trust-password=PASSWORD] [--trust-ca-certificates]
```
Comment (stgraber) [inline lxd/main_init.go]: I wouldn't make this a `lxd init` option.

Comment (stgraber) [reply lxd/main_init.go]: It's so uncommon a setup that it can definitely be enabled by hand through `lxc config`.

Comment (monstermunchkin) [reply lxd/main_init.go]: Fixed

File: doc/server.md (hunk 1 above)
Comment (monstermunchkin) [reply doc/server.md]: Done

**Pull request merged**